### PR TITLE
add POI view page

### PR DESCRIPTION
**Pull request by samlislalom**

### CODE REVIEW

File: src/components/CaseCard/CaseCard.tsx [hunk 1]
```diff
@@ -33,14 +33,35 @@ const useStyles = makeStyles( (theme: Theme) => ({
...
 }));
 
+
+
+
 const CaseCard = () => {
 
     const classes = useStyles();
     const history = useHistory();
     const inputHandler = () => {
```
Comment (samlislalom) [inline src/components/CaseCard/CaseCard.tsx:43]: rename

File: src/components/Header/CaseSubHeader.tsx [hunk 2]
```diff
@@ -58,9 +56,17 @@ const CaseSubHeader = (PoiInput : IPoiInput) => {
                 </Link>
                 <Typography className = {classes.currentItem}>{PoiInput.poi}</Typography>
             </Breadcrumbs>
-            <div className={classes.component}>
-                <Button className={classes.actionItem} startIcon={<AddIcon/>}>Add {PoiInput.poi}</Button>
-            </div>
+            <Grid 
+                container
```
Comment (samlislalom) [inline src/components/Header/CaseSubHeader.tsx:60]: text align to the right

File: src/components/POI/PoiCard.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+const PoiCard = (input : IPoiCard) => {
+    const classes = useStyles();
+    const poi = input.poi;
+    const time = new Date(Date.parse(poi.Created));
+    const getTime = (time : string) =>{
+        var dateTime = new Date(Date.parse(time));
+        return "Added: " + dateTime.getMonth() + "/" + dateTime.getDay() + "/" + dateTime.getFullYear();
+    }
+    return (
+        <div>
```
Comment (samlislalom) [inline src/components/POI/PoiCard.tsx:82]: try with Grid

File: src/components/POI/PoiCard.tsx [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+const PoiCard = (input : IPoiCard) => {
+    const classes = useStyles();
+    const poi = input.poi;
+    const time = new Date(Date.parse(poi.Created));
+    const getTime = (time : string) =>{
+        var dateTime = new Date(Date.parse(time));
+        return "Added: " + dateTime.getMonth() + "/" + dateTime.getDay() + "/" + dateTime.getFullYear();
+    }
+    return (
```
Comment (samlislalom) [inline src/components/POI/PoiCard.tsx:81]: new line before

File: src/components/POI/PoiContainer.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    }
+}));
+
+interface IPoisInput{
+    poiCollection : Poi[]
+    poiType: string
+}
+
+const currentCollection = (poiCollection : Poi[], poiType: string) =>{
+    return poiCollection.filter(poi=>poi.PoiType.toLowerCase() == poiType.toLowerCase());
```
Comment (samlislalom) [inline src/components/POI/PoiContainer.tsx:41]: ===

File: src/components/POI/PoiContainer.tsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+const PoiContainer = ({poiCollection, poiType} : IPoisInput) => {
+    const classes = useStyles();
+    return (
+        <div>
```
Comment (samlislalom) [inline src/components/POI/PoiContainer.tsx:47]: try Grid

File: src/API/Contracts.ts [hunk 7]
```diff
@@ -9,4 +7,24 @@ export interface CreateCaseObject {
...
         CellNumber?: string
     }
+}
+
+export interface Poi{
+
+    FirstName : string,
+    LastName : string,
+    Notes? : string,
+    PhoneNumber? : string,
```
Comment (samlislalom) [inline src/API/Contracts.ts:17]: try to rmb why

**Review by SlalomMarc — COMMENTED** [review]
Nicely done, sir!  Looks great.

**Pull request merged**